WalletWasabi.Tests/UnitTests/BranchAndBoundTests.cs

```diff
@@ -0,0 +1,97 @@
+using System.Collections.Generic;
+using System.Linq;
+using WalletWasabi.BranchNBound;
+using Xunit;
+
+namespace WalletWasabi.Tests.UnitTests
+{
+	/// <summary>
+	/// Tests for <see cref="BranchAndBound"/> class.
+	/// </summary>
+	public class BranchAndBoundTests
+	{
+		private Random Random { get; } = new();
+
+		[Fact]
+		public void RandomizedTest()
+		{
+			List<long> inputValues = GenerateList();
+			BranchAndBound selector = new(inputValues);
+			long target = 100000000;
```

Comment:
ultra nit: it is easier to read using separators 100_000_000.

Reply:
yeah, too many zeros.